tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -1,11 +1,126 @@
 jobs:
-- template: templates/win-ci.yml
-  parameters:
-    AgentPool : 'Win-CPU'
-    DoDebugBuild: 'true'
-    DoCompliance: 'false'
-    BuildCommand: '$(Build.SourcesDirectory)\tools\ci_build\build.py --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --cmake_path $(Build.BinariesDirectory)\cmake\bin\cmake.exe --ctest_path $(Build.BinariesDirectory)\cmake\bin\ctest.exe --use_tvm --use_automl --enable_pybind --use_mkldnn --use_openmp --build_shared_lib --build_csharp --enable_onnx_tests'
-    JobName: 'Windows_CI_Dev'
-    DoNugetPack:  'false'
-    NuPackScript : ''
-    DoTestCoverage: 'false'
+- job: 'build'
+  pool: 'Win-CPU-2019'
+  strategy:
+    maxParallel: 2
+    matrix:
+      debug:
+        BuildConfig: 'Debug'
+      release:
+        BuildConfig: 'RelWithDebInfo'
+  variables:
+    OrtPackageId: 'Microsoft.ML.OnnxRuntime'
+    MsbuildArguments: '-detailedsummary -maxcpucount -consoleloggerparameters:PerformanceSummary'
```

Comment:
If there's no cost to allowing customers to not upgrade their CRT, we should do it. This would go a long way in ensuring seamless upgrades in production. You mentioned that there's a flag to enable this. If there's a cost to enabling this flag, we can discuss is later.

Reply:
Yes. But as now we won't touch the release pipelines, it doesn't matter, we don't need the flag now. We can do it later.